Rubberduck.sln

```diff
@@ -250,6 +252,36 @@ Global
 		{40CC03E3-756C-4674-AF07-384115DEAEE2}.Release64|x64.Build.0 = Release|Any CPU
 		{40CC03E3-756C-4674-AF07-384115DEAEE2}.Release64|x86.ActiveCfg = Release|Any CPU
 		{40CC03E3-756C-4674-AF07-384115DEAEE2}.Release64|x86.Build.0 = Release|Any CPU
+		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I believe we can kill all legacy configurations. I've only ever used `Debug` and `Release` for over a year; the `xxxx64` configs have been useless since the unification of x86 and x64 installers IIRC. 

@ThunderFrame is `DebugAccess` of any use?

I'll make this PR clean up these noisy configurations.